### ISSUE TITLE
Hook problem when domain used 

### ISSUE THREAD

**Issue by jaumard** [balderdashy/sails#3012]
Hi,

I try to make a hook that load third party code. To prevent server to crash if third party crash I want to use node.js domain. But when I do and an error occurred, error is watched correctly but  hook doesn't load (timeout occurred) even if hook callback was called (and it was).
I also post it here http://stackoverflow.com/questions/30822351/sails-doesnt-start-if-hook-use-domain

And I make a default projet that reproduce the problem (don't forget to npm install after unzip). https://dl.dropboxusercontent.com/u/107371693/HookProblem.zip

Anyone have a idea of what append ?

**Comment by jaumard** [comment]
No one with this ? I'm always searching for a solution :(

**Comment by sailsbot** [comment]
Thanks for posting, @jaumard.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
